Godeps/Godeps.json

```diff
@@ -49,6 +50,10 @@
 			"ImportPath": "github.com/urfave/cli",
 			"Comment": "v1.19.1",
 			"Rev": "0bdeddeeb0f650497d603c4ad7b20cfe685682f6"
+		},
+		{
+			"ImportPath": "golang.org/x/sys/unix",
+			"Rev": "f3918c30c5c2cb527c0b071a27c35120a6c0719a"
```

Comment:
automatically added by godep, not sure if it should be here. But as unix also is called and depended by runtime-tools, I think may be it's OK